From ab6144a92adc0586071666d0ebf7c4d7f2fc0a72 Mon Sep 17 00:00:00 2001
From: Amelie Delaunay <amelie.delaunay@foss.st.com>
Date: Wed, 7 Feb 2024 11:44:11 +0100
Subject: [PATCH] arm64: Kconfig: expand STM32 Armv8 SoC with STM32MP21 SoCs
 family

Expand config ARCH_STM32 with the new STM32MP21 SoCs family which is
composed of STM32MP211, STM32MP213 and STM32MP215 SoCs.

Change-Id: I818cc57cb7457d8b9af0e9d8ef0d992f8b1fcc9c
Signed-off-by: Amelie Delaunay <amelie.delaunay@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/358026
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 arch/arm64/Kconfig.platforms | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/Kconfig.platforms b/arch/arm64/Kconfig.platforms
index 4c8507c030b3..407c87aada48 100644
--- a/arch/arm64/Kconfig.platforms
+++ b/arch/arm64/Kconfig.platforms
@@ -301,6 +301,8 @@ config ARCH_STM32
 			- STM32MP251, STM32MP253, STM32MP255 and STM32MP257.
 		- STM32MP23:
 			- STM32MP231, STM32MP233, STM32MP235.
+		- STM32MP21:
+			- STM32MP211, STM32MP213, STM32MP215.
 
 config ARCH_SYNQUACER
 	bool "Socionext SynQuacer SoC Family"
-- 
2.39.5

